java/cdk-environment/src/main/java/sleeper/environment/cdk/SleeperEnvironmentCdkApp.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.environment.cdk;
+
+import sleeper.environment.cdk.buildec2.BuildEC2Stack;
+import sleeper.environment.cdk.config.AppContext;
+import sleeper.environment.cdk.networking.NetworkingStack;
+import software.amazon.awscdk.App;
+import software.amazon.awscdk.Environment;
+import software.amazon.awscdk.StackProps;
+
+import static sleeper.environment.cdk.config.AppParameters.INSTANCE_ID;
+
+/**
+ * The {@link App} that sets up an environment for deploying Sleeper.
+ */
+public class SleeperEnvironmentCdkApp {
+
+    public static void main(String[] args) {
+        App app = new App();
+        Environment environment = Environment.builder()
+                .account(System.getenv("CDK_DEFAULT_ACCOUNT"))
+                .region(System.getenv("CDK_DEFAULT_REGION"))
```

Comment:
Just wondering whether it would be advantageous to use an instance properties file for account, region, networking stack name and build EC2 stack name?

Reply:
Can you think of a case where that would be useful? Note that the stack names are derived from the environment instance ID.

Reply:
It follows the pattern of https://github.com/gchq/sleeper/blob/8d8b0963c04cadfb345a2e5f9028691778eaaf8c/java/system-test/src/main/java/sleeper/systemtest/cdk/SystemTestApp.java#L93 and https://github.com/gchq/sleeper/blob/8d8b0963c04cadfb345a2e5f9028691778eaaf8c/java/cdk/src/main/java/sleeper/cdk/SleeperCdkApp.java#L228